### ISSUE TITLE
FIDO U2F hardware key support (2FA)

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#1223]
This is maybe a bit too cutting edge, but I have started using U2F keys myself for 2-factor (2FA) logins. It's pretty new feature, at the moment only fully supported by Chrome, but there is a plugin and a WIP version working in Firefox nightly.

Google Authenticator support (OTP?) is a separate issue (#220). 

Opening this issues so notes can be collected somewhere.

**Comment by povrazor** [comment]
For anyone that is curious about this

* The standard was defined by Yubico and Google.
* If you're a Chrome (or Opera) user, you're good. If not, you might want to hold off on this.
* Yubico makes keys, but they're perhaps the most expensive, costing $20-$50, BUT there is a tiny key that disappears in to a USB socket. Perfect for a laptop. :heart: 
* The HyperFido Mini is another good one. I got two for $10 each (CAD). Good for a desktop, but it does have a bright LED on it so you may want to put it on the back of your PC, in a port that powers off.
* More sites supporting it: http://www.dongleauth.info/

**Comment by wmpedersen** [comment]
I'm trying to move away from SMS and switching to Authenticator/U2F. So far I've done U2F on everything that I can, Unfortunately, that's only maybe 5 services for U2F.

**Comment by povrazor** [comment]
Yeah choices are definitely limited. I can appreciate that Google and all the notable source code hosts are on the ball here, but I would certainly like to see more. Incomplete Firefox and no Edge support doesn't help.
